samples/samples-js/AddProductWithIdentityColumnIncluded/index.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const itemToInsert = {
```

Comment:
You should be parsing the name and cost from the parameters like the C# function